packages/react-component-library/src/components/ClassificationBar/ClassificationBar.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { ComponentWithClass } from '../../common/ComponentWithClass'
+import { StyledClassificationBar } from './partials/StyledClassificationBar'
+
+export interface ClassificationProps extends ComponentWithClass {
+  isSecret?: boolean
+}
+
+export const ClassificationBar = ({
+  isSecret,
+  ...rest
+}: ClassificationProps) => {
+  const classificationText = isSecret ? 'Secret' : 'Official Sensitive'
+  return (
+    <StyledClassificationBar
+      data-testid="classification-bar"
+      isSecret={isSecret}
```

Comment:
styled-components has a convention called transient props.

This helps to prevent polluting the DOM output.

https://styled-components.com/docs/api#transient-props

We have some recommended practices related to styled-components here in out contributing guidelines:

https://github.com/Royal-Navy/design-system/blob/master/docs/contributing.md#styled-components

Reply:
Ok cool, just had a read and got my head round this. Makes sense!